test/e2e/e2e.go

```diff
@@ -223,7 +223,7 @@ func checkObject(t *testing.T, expected interface{}, actual interface{}) bool {
 }
 
 func VerifyAllOperationsSucceeded(t *testing.T, clients []*client.FireFlyClient, startTime time.Time) {
-	tries := 3
+	tries := 30
 	delay := 2 * time.Second
```

Comment:
60 seconds seems quite long

Reply:
I'm happy to reduce - I think I need 30seconds with 2 confirmations, obviously more with more confirmations

Reply:
OK, if we're sure 30 seconds is "happy path" in this world, 60 seems reasonable. It does make the suite take quite a long time then, right?

Reply:
Decision: Leave at 60 for now